gradle/libs.versions.toml

```diff
@@ -36,7 +36,7 @@ slf4j-lambda = "kr.pe.kwonnam.slf4j-lambda:slf4j-lambda-core:0.1"
 slf4j-simple = "org.slf4j:slf4j-simple:1.7.36"
 vavr = "io.vavr:vavr:0.10.4"
 # Plugins
-pluginPackages-aspectj-postCompileWeaving = "io.freefair.gradle:aspectj-plugin:6.5.1"
+pluginPackages-aspectj-postCompileWeaving = "io.freefair.gradle:aspectj-plugin:6.6"
```

Comment:
This should happen automagically at midnight UTC anyway... or is there a specific reason to perform this update manually? 🤔

Reply:
maybe the release happend after the automagical update on ci